### PR TITLE
Added greater intellect enchant for bracers

### PR DESCRIPTION
**Pull request by RyanFehr**
This PR adds enchant bracers - Greater Intellect as it is bis bracer enchant for many classes. All classes which have it as bis have it added as an option now.

### CODE REVIEW

File: Class/Common/Enchants/EnchantStatic.cpp [hunk 1]
```diff
@@ -199,6 +202,9 @@ EnchantStatic::~EnchantStatic() {
     case EnchantName::EnchantWeaponSpellPower:
         pchar->get_stats()->decrease_base_spell_damage(30);
         break;
+    case EnchantName::EnchantBracerGreaterIntellect:
+        pchar->get_stats()->increase_intellect(7);
```
Comment (timhul) [inline Class/Common/Enchants/EnchantStatic.cpp]: This needs to be decrease_intellect(), as it currently keeps stacking intellect bonus when applying enchant and removing enchant.

Comment (RyanFehr) [reply Class/Common/Enchants/EnchantStatic.cpp]: Fixed

**Comment by RyanFehr** [comment]
> Please squash your commits into a single commit (I think it will still be of an appropriate size) and I'll merge this.
> 
> Thanks for the pull request!

You should be able to do a squash and merge when doing the merge into master and that will ensure the PR shows up as one commit rather than 3.
image

**Comment by timhul** [comment]
> You should be able to do a squash and merge when doing the merge into master and that will ensure the PR shows up as one commit rather than 3.

I was under the impression it would do a merge rather than rebase so I have it disabled for this project, but after checking [this](https://help.github.com/en/github/administering-a-repository/about-merge-methods-on-github) it appears I'm mistaken. Thanks for the info!

**Pull request merged**